src/main/java/com/bookstore/service/BookService.java

```diff
@@ -10,4 +10,8 @@ public interface BookService {
     List<BookDto> findAll();
 
     BookDto findById(Long id);
+
+    BookDto update(Long id, CreateBookRequestDto bookRequestDto);
+
+    void delete(Long id);
```

Comment:
Does your Checkstyle plugin work correctly? I see that you don’t have an empty line at the end of the file in this class. Please check it ;)

Reply:
<img width="1512" alt="Screenshot 2024-10-24 at 19 38 22" src="https://github.com/user-attachments/assets/4c83e2e1-2999-4109-9970-5f5092a1f4f3">
I don't understand anything again. What exactly is the error with this plugin?